core/plugins/register_plugins.cpp

```diff
@@ -0,0 +1,35 @@
+#include "NvInferPlugin.h"
+#include "NvInferPluginUtils.h"
+#include "core/plugins/impl/interpolate_plugin.h"
+#include "core/plugins/impl/normalize_plugin.h"
+#include "core/plugins/plugins.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace plugins {
+namespace impl {
+
+// Main registry for all flavours of plugins (eg: TRT plugins, TRTorch opensourced plugins)
+class TRTorchPluginRegistry {
+ public:
+  TRTorchPluginRegistry() {
+    auto trtorch_logger = util::logging::TRTorchLogger(
+        "[TRTorch Plugins Context] - ",
+        util::logging::get_logger().get_reportable_severity(),
+        util::logging::get_logger().get_is_colored_output_on());
+    // register libNvInferPlugins and TRTorch plugins
+    initLibNvInferPlugins(&trtorch_logger, "");
+    REGISTER_TRTORCH_PLUGIN(InterpolatePluginCreator);

```

Comment:
Can we just have the Register calls at the bottom of their respective source files? I think we only need the static object for the logger and the nvinfer library. Also we should probably hold the trtorch_logger as a member variable so it only gets destroyed at the end.

Reply:
I'll move the logger object as a member variable. 
When I compiled the REGISTER calls with their source files, the `libtrtorchrt` was unable to find the plugin creators. When I moved it to this `register_plugins.cpp`, I could see them getting registered correctly. I shall double check.

Reply:
I moved it to respective source files and tested it and it seemed to work now to my surprise. When I tested it a couple of days back, it did not. I'd appreciate if you can try this out on your system just to validate the behavior. The way I currently test this is by building TRTorch tar file and running `sample_rt_app`